account_journal_lock_date/models/account_journal.py

```diff
@@ -1,19 +1,17 @@
 # -*- coding: utf-8 -*-
-# Copyright 2017 ACSONE SA/NV
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import api, fields, models
+from openerp import models, fields, api
+from openerp.exceptions import ValidationError, UserError
 
 
 class AccountJournal(models.Model):
-
     _inherit = 'account.journal'
 
     journal_lock_date = fields.Date(
-        string="Lock date",
+        name='Lock Date',
```

Comment:
`name` instead of `string` is a mistake, it's fixed. I capitalized _Date_ for consistency with the rest of the code base. But now that I think of it, i might mess up translation. I'll leave it as it is.